### DISCUSSION TITLE
Commits being tagged by an older user

### DISCUSSION THREAD

**Discussion by OkanArikan** [community/community#22818] [New to GitHub]
<p>I had an account with MFA turned on with my computers SSH key registered.</p>
<p>Of course I did not download recovery codes and of course I upgraded my phone without backing up the authenticator app :frowning:</p>
<p>I now have a new account :slight_smile: but for some reason all the commits coming from my computer are being authored by my old username (I’m using GitHub desktop). I see my old user name’s icon next the commit box but I cannot change this (yes, I’m logged into my new account). Has anybody ran into this issue?</p>
<p>Sample PR from my new account with commit from my old: <a href="https://github.com/CedricGuillemet/ImGuizmo/pull/129" rel="noopener nofollow ugc">https://github.com/CedricGuillemet/ImGuizmo/pull/129</a></p>

**Comment by steveward** [comment, marked as answer]
<p>Welcome <span class="mention">@OkanArika</span>.  You can check the name and email address that are being used in your .gitconfig by going to <code>File</code> &gt; <code>Option</code> &gt; <code>Git</code> in GitHub Desktop. Make sure the email address being used there matches an email address that is associated with your currently logged in GitHub account (<a href="https://github.com/settings/emails">https://github.com/settings/emails</a>). You can read more about how commit attribution works here:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/github/committing-changes-to-your-project/why-are-my-commits-linked-to-the-wrong-user" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="16" height="16">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/github/committing-changes-to-your-project/why-are-my-commits-linked-to-the-wrong-user" target="_blank" rel="noopener">Why are my commits linked to the wrong user? - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>We’re hoping to make some changes to GitHub Desktop in the future to detect when these situations occur (see <a href="https://github.com/desktop/desktop/issues/610">https://github.com/desktop/desktop/issues/610</a> for the tracking issue).</p>
